spec/rails_helper.rb

```diff
@@ -29,7 +29,11 @@
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
-
+  # The following line allows you to call FactoryGirl methods while omiting the
+  # prepending FactoryGirl.
+  # For instance you can type create(:piece) instead of
+  # FactoryGirl.create(:piece)
+  config.include FactoryGirl::Syntax::Methods
```

Comment:
This makes it so you don't have to type FactoryGirl before every build or create method.